src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -48,11 +48,11 @@
   -->
 
   <xsl:template name="pickfont-sans">
-    <xsl:text>Arial,sans-serif</xsl:text>
+    <xsl:text>Liberation Sans,Arial,sans-serif</xsl:text>
   </xsl:template>
 
   <xsl:template name="pickfont-serif">
-    <xsl:text>Georgia,serif</xsl:text>
+    <xsl:text>DejaVu Serif,Liberation Serif,serif</xsl:text>
```

Comment:
You are preaching to the choir with regards to the use of proprietary fonts. That's why we switched to open source fonts in the default Asciidoctor stylesheet for HTML. (There's a whole discussion around the decision to switch that echos many of the points you made here).

The problem in this PR is that the font choices you added aren't available by default on all platforms (DejaVu, Liberation, etc). So the output is going to vary based on which platform you run this on. That's actually already true, but this change will cause the output to change for people who currently rely on this script (in a different, inconsistent way).

If we really want to align with the default Asciidoctor stylesheet for HTML, we should be using NotoSerif and OpenSans. In fact, we could add those fonts to this project and point to them so that the output is always the same regardless of the platform. I think that's the right next step.